### PR TITLE
Unity tool ToolPath changes

### PR DESCRIPTION
**Pull request by jonsth131**
Updated UnityTasks.GetToolPath to find default Windows x86/x64 installation paths.

### CODE REVIEW

File: source/Nuke.Common/Tools/Unity/UnityBaseSettings.cs [hunk 1]
```diff
@@ -10,11 +10,13 @@ namespace Nuke.Common.Tools.Unity
 {
     public partial class UnityBaseSettings
     {
+        public override string ToolPath { get; internal set; } = UnityTasks.GetToolPath();
```
Comment (matkoch) [inline source/Nuke.Common/Tools/Unity/UnityBaseSettings.cs]: why this change?

Comment (jonsth131) [reply source/Nuke.Common/Tools/Unity/UnityBaseSettings.cs]: This change was unnecessary, reverted the last commit.
